README.md

```diff
@@ -28,7 +28,7 @@ browser-like environment) via `<script>` tags. You'll need to include the
 formiojs library in your document _before_ formio-sfds, like so:
 
 ```html
-<script src="https://unpkg.com/formiojs@4.11.2/dist/formio.full.min.js"></script>
+<script src="https://unpkg.com/formiojs@4.11.2/dist/formio.form.min.js"></script>
 <script src="https://unpkg.com/formio-sfds@7.x/dist/formio-sfds.standalone.js"></script>
```

Comment:
If we always want the patches to be included with the Form.io code, seems like we should include it in our own package, so there's just one http request and it's always the correct version.  If they're separate, the versions can get out of sync, or one of the script tags might get forgotten.

Reply:
Also, how do these script tags get included in the sf.gov pages?  Does making a change here automatically push them to the site?